packages/cli-test/src/cli/cli-process.ts

```diff
@@ -15,6 +15,13 @@ export interface SlackCLIGlobalOptions {
    * `qa` and `apihost` will both supersede this option.
    */
   dev?: boolean;
+  /** @description Ignore warnings and continue executing command. Defaults to `true`. */
+  force?: boolean;
+  /**
+   * @description Application instance to target. Can be `local`, `deployed` or an app ID string.
+   * Defaults to `deployed`.
+   */
+  instance?: string;
```

Comment:
Since this maps to the `--app` flag, perhaps `instance` is not the right name for this option? WDYT?